Test_alerter.py

```diff
@@ -0,0 +1,8 @@
+import alerter
+alert_failure_count = 0
+Threshold_temp = 100
+alerter.alert_in_celcius(400.5,alerter.network_alert_stub)
+alerter.alert_in_celcius(303.6,alerter.network_alert_stub)
+assert alert_failure_count == 2
```

Comment:
Need to consider 3 min values, i.e. <threshold, == threshold and >threshold, and then assert required expectation.